web/src/main/java/org/cbioportal/web/ClinicalDataEnrichmentController.java

```diff
@@ -39,6 +39,7 @@
 
 @InternalApi
 @RestController
+@RequestMapping("/api")
```

Comment:
if there's a way to configure this for e.g. all Controllers or e.g. `InternalApi` and `PublicApi` groups that might be better but no worries if not